src/main/resources/static/css/application.css

```
@@ -1,3 +1,12 @@
+/*
+Darkest-Blue: #062844
+Medium-Blue: #064172
+Darker-Blue: #1D77C3
+Lightest-Blue: #42BFDD
+*/
```

There's a really cool feature in css, called css custom properties. Instead of keeping the colours in a comment and copy/pasting, we can keep one source of truth, and reuse variables throughout the css.

```suggestion
:root {
  --darkest-blue: #062844;
  --medium-blue: #062844;
  --darker-blue: #1d77c3;
  --lightest-blue: #42bfdd;
}
```

and then we can use the varables in other elements like so:
```
body {
  background: var(--lightest-blue)
}

```

Alternatively, we can add a bit of semantics and remove the association with the colour itself like so (ignore the colours)
```
:root {
  --accent: #062844;
  --accent-darker: #062844;
  --accent-dark: #1d77c3;
  --accent-light: #42bfdd;
  
  --background-light: #230123;
  --background: #ab32ff;
  
  --primary-color: #111111;
}
```

I do this a lot with my personal web projects, e.g. https://github.com/AndrewYHuang/love-jar/blob/main/src/style.css